web/package.json

```diff
@@ -30,6 +30,7 @@
     "react-hook-form": "^7.27.1",
     "react-icons": "^4.6.0",
     "react-markdown-editor-lite": "^1.3.2",
+    "react-router-dom": "^6.26.2",
```

Comment:
Remove packages if they aren't used since they have the potential to cause problems later